Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+FROM marcbachmann/libvips:latest
```

Comment:
Can we not do:
```
FROM ubuntu:16.04
apt-get update
apt-get install golang-1.8-go  #https://github.com/golang/go/wiki/Ubuntu
```
It may requires to add a ppa